docs/developer-docs/latest/plugins/sentry.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Sentry - Strapi Developer Docs
+description: Send email from your server or externals providers.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/plugins/email.html
+---
+
+# Sentry
+
+This plugin enables you to track errors in your Strapi application using [Sentry](https://sentry.io/welcome/).
+
+By using the Sentry plugin you can:
+
+* Initialize a Sentry instance upon startup of a Strapi application
+* Send Strapi application errors as events to Sentry
+* Include additional metadata in Sentry events to assist in debugging
+* Expose a [global Sentry service](#global-sentry-service)
+
+## Installation
+
+Install the Sentry plugin by adding the dependency to your Strapi application as follows:
+
+<code-group>
+
+<code-block title="NPM">
+```sh
+npm install @strapi/plugin-sentry
+```
+</code-block>
+
+<code-block title="YARN">
+```sh
+yarn add @strapi/plugin-sentry
+```
+</code-block>
+
+</code-group>
+
+Then you must [configure](#configuration) the plugin to enable your Strapi application to send events to the Sentry instance.
+
+## Configuration
+
+Create or edit your `./config/plugins.js` file to configure the Sentry plugin. The following properties are available:
+
+| Property | Type | Default Value | Description |
+| -------- | ---- | ------------- |------------ |
+| `dsn` | string | `null` | Your Sentry [data source name](https://docs.sentry.io/product/sentry-basics/dsn-explainer/). |
+| `sendMetadata` | boolean | `true` | Whether the plugin should attach additional information (e.g. OS, browser, etc.) to the events sent to Sentry. |
+| `init` | object | `{}` | A config object that is passed directly to Sentry during initialization. See the official [Sentry documentation](https://docs.sentry.io/platforms/node/configuration/options/) for all available options. |
+
+An example configuration:
+
+```js
+// path: ./config/plugins.js
+
+module.exports = ({ env }) => ({
+  // ...
+  sentry: {
+    enabled: true,
+    config: {
+      dsn: env('SENTRY_DSN'),
+      sendMetadata: true,
+    },
+  },
+  // ...
+});
+```
+
+### Environment configuration
+
+Using the `env` utility, you can enable or disable the Sentry plugin based on the environment. For example, to only enable the plugin in your `production` environment:
+
+```js
+// path: ./config/plugins.js
+
+module.exports = ({ env }) => ({
+  // ...
+  sentry: {
+    enabled: env('NODE_ENV') === 'production',
+  },
+  // ...
+});
+```
+
+## Global Sentry service
```

Comment:
Shouldn't we add a verb here to understand better what this section is about? Something like "Global Sentry service access"/"Accessing Global Sentry service" (depending on the title format you'll choose) would be more precise for readers in my opinion. What do you think?